docs/software/meshtasticator/interactive-sim.mdx

```diff
@@ -0,0 +1,127 @@
+---
+id: interactive-sim
+title: Interactive Simulator Usage Guide
+sidebar_label: Interactive Event Simulator
+sidebar_position: 2
+description: Interactive Simulator usage guide for simulating multiple Meshtastic instances with TCP-based communication and configurable pathloss models.
+---
+
+import ReactPlayer from "react-player";
+
+The Python script `_interactiveSim.py_` uses the [Linux native application of Meshtastic](https://meshtastic.org/docs/software/linux-native) to simulate multiple instances of the device software. These instances communicate using TCP via the script, simulating the LoRa chip. The simulator forwards messages from the sender to all nodes within range, based on their simulated positions and the selected pathloss model (see [Pathloss Model](#pathloss-model)). **Note:** Packet collisions are not yet simulated.
```

Comment:
It seems these underscores cannot be used inside ` `.

Reply:
Think you just have to escape them
`\_interactiveSim.py\_`